### DISCUSSION TITLE
Next release date?

### DISCUSSION THREAD

**Discussion by YotYot** [pyinstaller/pyinstaller#5986] [Features & Development]
Hey, Is there any visibility for the next release of pyinstaller? 

I'm waiting for the PyQt6 support :)

Thanks, Yotam

**Comment by bwoodsend** [comment]
@Legorooj Now!

**Reply by Legorooj** [reply]
Will release on Sunday.

**Comment by Legorooj** [comment]
v4.4 has been released :tada:.

**Reply by sha-trix** [reply]
4.5 release date?

**Reply by bwoodsend** [reply]
The plan is this weekend but things rarely do go to plan...
